src/dmd/dmangle.d

```diff
@@ -1,4 +1,9 @@
 /**
+ * Does name mangling for D symbols, which generates a unique name for a linker symbol based
```

Comment:
Too long, and inconsistent with the `cppmangle` side which does not explain what mangling is.
But really, I would expect anyone working on a compiler to know that. We can still add a `See_Also` to wikipedia if you feel like it, but we shouldn't paraphrase it.